ghcide/src/Development/IDE/GHC/Compat/Core.hs

```diff
@@ -491,7 +491,12 @@ import           GHC.Unit.Module              hiding (ModLocation (..), UnitId,
                                                toUnitId)
 import qualified GHC.Unit.Module              as Module
 import           GHC.Unit.State               (ModuleOrigin (..))
-import           GHC.Utils.Error              (Severity (..), emptyMessages)
+import           GHC.Utils.Error              (Severity (..), emptyMessages
+#if MIN_VERSION_ghc(9,3,0)
+                                              , mkPlainErrorMsgEnvelope)
+#else
+                                              )
+#endif
```

Comment:
Nitpick: a way to make the CPP if not have the else branch :slightly_smiling_face: 
```suggestion
                                              , mkPlainErrorMsgEnvelope
#endif
                                              )
```